web3.js/scripts/typegen.sh

```diff
@@ -7,13 +7,16 @@ npx tsc -p tsconfig.d.json -d
 npx rollup -c rollup.config.types.js
 
 # Replace export with closing brace for module declaration
-sed -i '$s/export {.*};/}/' lib/index.d.ts
+sed -i.bak '$s/export {.*};/}/' lib/index.d.ts
```

Comment:
nit: looks like there's typically a space before the extension
```
     sed [-Ealnru] command [-I extension] [-i extension] [file ...]
```

Reply:
Thoughtbot warned me against leaving a space:

> Note that on Linux systems, a space after -i might cause an error, so instead of -i .bak, you can try -i.bak.

https://thoughtbot.com/blog/sed-102-replace-in-place